crates/driver/src/infra/config/file/load.rs

```diff
@@ -18,9 +18,8 @@ pub async fn load(network: &blockchain::Network, path: &Path) -> infra::Config {
     let data = fs::read_to_string(path)
         .await
         .unwrap_or_else(|e| panic!("I/O error while reading {path:?}: {e:?}"));
-    // Not printing detailed error because it could leak private keys.
     let config: file::Config = toml::de::from_str(&data)
-        .unwrap_or_else(|_| panic!("TOML syntax error while reading {path:?}"));
+        .unwrap_or_else(|err| panic!("TOML syntax error while reading {path:?} {err:?}"));
```

Comment:
I am personally against this change - this assumes that we will only use KMS in our infrastructure which may not be the case (For example, when trying out a deployment on a new network, or if a `driver` is run by an external solver). Note that the private key can be in this configuration and be leaked by this error (which is, IMO not OK).

I suggest adding a `TOML_TRACE_ERROR` environment variable to control displaying additional panic information to facilitate debugging, but not have this be the default behavior.

```suggestion
        .unwrap_or_else(|err| if env::var("TOML_TRACE_ERROR").as_deref() == Ok("1") {
            panic!("TOML syntax error while reading {path:?} {err:?}")
         } else {
            panic!("TOML syntax error while reading {path:?}")
         });
```

I think this has bitten us before in the past, and once a PK is leaked there is no going back.